VimeoNetworking/Sources/AuthenticationController.swift

```diff
@@ -82,13 +82,13 @@ final public class AuthenticationController
      
      - returns: a new `AuthenticationController`
      */
-    public init(client: VimeoClient)
+    public init(client: VimeoClient, appConfiguration: AppConfiguration)
     {
-        self.configuration = client.configuration
+        self.configuration = appConfiguration
         self.client = client
-        self.accountStore = AccountStore(configuration: client.configuration)
+        self.accountStore = AccountStore(configuration: configuration)
         
-        self.authenticatorClient = VimeoClient(appConfiguration: client.configuration)
+        self.authenticatorClient = VimeoClient(appConfiguration: configuration)
```

Comment:
Ah ok this was pointed out by @mikew-personal - why would we not just pass the same client into the authenticator. Seems like we should be able to do `self.authenticatorClient = client` here. I know this was already in place before your changes @fyell but any thoughts on this?